packages/network-of-terms-catalog/catalog/queries/search/stcn-drukkers.rq

```diff
@@ -16,9 +18,17 @@ WHERE {
     ?uri ?predicate ?label .
     VALUES ?predicate { rdfs:label schema:name schema:alternateName }
     ?label <bif:contains> ?virtuosoQuery .
-
+    # STCN defines multiple URIs for the same printer.
+    # The distinction between these URIs is based on the place where the printer lived and worked.
+    # See for example "Plantijn"
+    # http://data.bibliotheken.nl/id/thes/p075556251 for Plantijn working in Leiden
+    # http://data.bibliotheken.nl/id/thes/p338012834 for Plantijn working in Antwerp
+    # The original source (STCN) shows also a time period, this data currently not available in the LOD version.
+    OPTIONAL { 
+     ?uri schema:location/schema:address/schema:addressLocality ?place 
+    }
+    BIND(COALESCE(IF(BOUND(?place), ?place, ?noPlace)) as ?scopeNote)
```

Comment:
Couldn’t this be simplified to:

```suggestion
    OPTIONAL { 
        ?uri schema:location/schema:address/schema:addressLocality ?scopeNote
    }
```

Reply:
Good point, a left over of a more complex approach, removed the BIND statement